src/scripts/index.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Utility scripts to improve the experience if JS is enabled
+ */
+
+import { MeetupApi } from "./meetupApi";
+
+if ("Promise" in window) {
+  void (function main() {
+    const elementIsAnchor = (
+      element: HTMLElement | null
+    ): element is HTMLAnchorElement => {
```

Comment:
Wouldn't it be easier to call `element instanceof HTMLAnchorElement`? 🤔

Reply:
Fair point. I've entered "Don't crash on IE mode'.